kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/marketplace/presenter/MarketplacePresenter.java

```diff
@@ -46,6 +52,12 @@
 
 	private Observer<Offer> pendingOfferObserver;
 	private Observer<Order> completedOrderObserver;
+
+	private ObservableData<Boolean> isEarnEmpty = ObservableData.create();
+	private ObservableData<Boolean> isSpendEmpty = ObservableData.create();
```

Comment:
default value?

Reply:
shouldn't be, should be set on the first time we are setting offers